umbra-js/src/abi/UmbraBatchSend.sol/UmbraBatchSend.json

```diff
@@ -0,0 +1,4223 @@
+{
+  "abi": [
```

Comment:
This is the full output artifact, which includes wayyy more data than we need which is why it's so big. Let's only include the `abi` section.

Even better would be to take the abi and convert it to "human readable form" in `constants.ts`, like we do for other ABIs in that file. You can do this by creating an ethers.js interface from the ABI and and then using the interface's `format` method. Check out the [docs](https://docs.ethers.org/v5/single-page/#/v5/api/utils/abi/interface/) for more info

Reply:
Awesome! Addressed here: https://github.com/ScopeLift/umbra-protocol/pull/426/commits/f09f9b371633493016d6577c2fea0b2e02ab8d16

Reply:
Can we move it to `constants.ts` where the other non-Umbra.sol ABIs are? In a separate PR we should move the Umbra ABI declaration there too.

Also, CI is failing which it seems is because we still try importing the old (non-existent) ABI file in tests instead of using the new ABI declaration

Reply:
Addressed by your PR: https://github.com/ScopeLift/umbra-protocol/pull/426/commits/8dae74b4730f11ba3d34204707f81ed9742bafa7#diff-4425693d92f00c46b6f05b597ec8880c030e8575db36e43e23d26e956e573901